README.md

```diff
@@ -1171,13 +1172,14 @@ Bear in mind that with the introduction of the new V8 engine alongside the new E
 
 <br /><br /><br />
 
-## ![✔] 8.10. Prefer smaller images
+## ![✔] 8.10. Prefer smaller Docker base images
```

Comment:
Consider mentioning also Slim which is popular like Alpine. They don't mitigate these issues as they also have CVEs but they reduce the likelihood of vulnerabilities and compact the attack surface